pkg/ccloudv2/networking.go

```diff
@@ -36,3 +36,33 @@ func (c *Client) UpdateNetwork(envId, id string, updateReq networkingv1.Networki
 	resp, httpResp, err := c.NetworkingClient.NetworksNetworkingV1Api.UpdateNetworkingV1Network(c.networkingApiContext(), id).NetworkingV1NetworkUpdate(updateReq).Execute()
 	return resp, errors.CatchCCloudV2Error(err, httpResp)
 }
+
+func (c *Client) ListNetworks(envId string) ([]networkingv1.NetworkingV1Network, error) {
+	var list []networkingv1.NetworkingV1Network
+
+	done := false
+	pageToken := ""
+	for !done {
+		page, err := c.executeListNetworks(envId, pageToken)
+		if err != nil {
+			return nil, err
+		}
+		list = append(list, page.GetData()...)
+
+		pageToken, done, err = extractNextPageToken(page.GetMetadata().Next)
```

Comment:
add a test case to test the list operation with multi-page response (see if you can mock the page size to 1 or something).

Reply:
hmm, I am not sure how to mock the page size to 1. @brianstrauch Is it something that we can do with our current setup?

Reply:
You should be able to add a next page token here, right? https://github.com/confluentinc/cli/blob/16fa2f4f9a93d06cdc7dfc6b58474fa5ba2558a7/test/test-server/networking_handlers.go#L165

Reply:
Thanks @brianstrauch! @ejagade I updated the networking_handlers to mock a multi-page response.